### PR TITLE
Fixes #1148 App not taking API_HOST env all the time

### LINKED ISSUE #1148: App not taking API_HOST env all the time
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**

Currently, in development the app doesn't take env `API_HOST` everytime.

**Expected behavior:**

Make the app recieve all its APIs from env API_HOST by exporting the env in system wide mode.

**Steps to reproduce:**

**Related code:**

```
insert any relevant code here else remove this section
```

**Other information:**

**System information:** 

```
Your operating system
```

```
output of `node -v`
```

```
output of `npm ls ember-cli`
```

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by bhaveshAn**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Apply system wide env API_HOST

#### Changes proposed in this pull request:

- Doc update to apply system wide env API_HOST
-
-

Fixes #1148 App not taking API_HOST env all the time

@ritikamotwani @pradeepgangwar @CosmicCoder96 @niranjan94 Please review thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,16 @@ If you face error in `npm install` for node and npm version, then install node v
 * `ember serve`
 * Visit your app at [http://localhost:4200](http://localhost:4200).
 
+### Note:
+If someone finds problem to run the app decoupled with [backend](https://github.com/fossasia/open-event-server) locally.
```
Comment (CosmicCoder96) [inline README.md]: The app is always decoupled from the backend, this statement is not valid.

File: README.md [hunk 2]
```diff
@@ -53,6 +53,16 @@ If you face error in `npm install` for node and npm version, then install node v
...
+Follow the steps:
+1. `sudo gedit /etc/environment`
```
Comment (CosmicCoder96) [inline README.md]: `gedit`  makes it too speific. Let;s leave the editing to the user.

File: README.md [hunk 3]
```diff
@@ -53,6 +53,18 @@ If you face error in `npm install` for node and npm version, then install node v
 * `ember serve`
 * Visit your app at [http://localhost:4200](http://localhost:4200).
 
+### Note:
+For using the local deployment of server along with the frontend, the environment variable API_HOST must be passed accordingly. Follow the steps:
```
Comment (mayank8318) [inline README.md]: *Follow these steps:* IMO

**Comment by niranjan94** [comment]
@bhaveshAn doesn't the `.env` file aim to accomplish the same ?

**Comment by CosmicCoder96** [comment]
@Kreijstal I agree with @niranjan94 as there is no need to do this at all this if `.env` file is present for the sole purpose of exporting environment variables.

**Comment by niranjan94** [comment]
@bhaveshAn I'm closing this PR and issue. Maybe you can add more details to the README about the `.env` file.

**Comment by niranjan94** [comment]
I use `ember-cli-dotenv` in a lot of ember projects and haven't faced that issue.
Maybe, check https://github.com/fivetanley/ember-cli-dotenv and ensure `ember-cli-dotenv` is configured correctly. (If not, open a PR for that). We have this library just for the purpose of setting the correct env variables. Lets make sure we use that. 

If you still think there is some issue, open an issue at https://github.com/fivetanley/ember-cli-dotenv/issues with the correct steps to reproduce. (_if possible, you could even send a patch to that library 😉_)

**Comment by pradeepgangwar** [comment]
@bhaveshAn I checked today and found that i actually specified API_HOST as 127.0.0.1:5000 in .env that's why it used to take that value by default. I am sure .env file works as expected. :sweat_smile:

**Pull request closed without merging**